### ISSUE TITLE
MLOops: provide RSS feed

### ISSUE THREAD

**Issue by zwpaper** [tensorchord/envd-docs#261]
I found the MLOops podcast is available on 小宇宙 and Spotify podcast, It is not convenient for RSS users.

Can we also provide the RSS feed, so that users can import to general podcast apps

**Comment by gaocegege** [comment]
Yeah, sure. I think we can maintain a website for the podcast too.

**Comment by gaocegege** [comment]
https://anchor.fm/s/d97a63c0/podcast/rss

**Comment by zwpaper** [comment]
@gaocegege looks like there's something wrong with the rss feed? It only has one hour long and doesn't look like ended normally

**Comment by kemingy** [comment]
> @gaocegege looks like there's something wrong with the rss feed? It only has one hour long and doesn't look like ended normally

Yep. It's 1 hour and it ends up suddenly. I think we can add an ending remark.
